README.md

```diff
@@ -1 +1,190 @@
+
+Note that in the container we use the apache URL rewrite. That needs to be installed first on the machine if you don't want to use the container approach. 
+
+**Containerize folder**
+
+This folder contains helper files for the image build. 
+
+- containerize folder content 
+  * "apache" folder, contains the apache configurations for URL rewrite and a web-root** folder, more information [here](#URL-redirect)
+  
+**Note that the CERN specific logos and backgrounds are not included in the repository. These were originally under the "containerize/configuration/web-root/images". In case you want to build your container you need to create this folder and add images with the following names:
+* opendays_background.png => background image of the application
+* Site_web_logo.png =>  banner top left CERN Opendays reservation system logo 
+* Site_web_opendays.png =>  banner top right CERN officiel site text logo
+
+Currently there is an empty.txt for place holding purpose there. Thise file is removed during the container build process.
+
+These files were hosted on the same level (top) as the language selector page. In case you don't provide these files you will see broken image icon in the browser once you load the application.
+
+- scripts
+
+Start up script which starts when container starts and runs the httpd as a foreground process.
+
+# How to use the container
+
+To run the image you need to have [Docker installed](https://docs.docker.com/install/).
+
+Once your environemnt is ready you update the settings in the ./Dockerfile. 
+Note that you need to use a RedHat or CentOS image. You can use for example the docker hub image : "centos:centos7"
+
+From the root folder, where the Dockerfile is run the following command.
+
+```
+>docker build -t reservation-system-front-end -f ./Dockerfile .
+```
+
+To run the image you need to forward the port 
+
+```
+>docker run -it -p 80:80 --rm --name reservation-front-end reservation-system-front-end
+```
+
+You access your image via http://localhost
+
+# Langauge preference selector application
+
+For the language selector page first we went for a non-Angular solution, but during the testing we noticed that certain browsers on phones zoom the top banner for the angular applications. In order to have the same banner size we decided to create an individual application. This application you can find under the application-welcome folder. The built application was hosted at the root level of the web server.
```

Comment:
mmm, not sure if this is the right place for this explanation :)

Reply:
Discussed, we leave it like this